### PR TITLE
Added prometheus mysql exporter chart

### PR DESCRIPTION
**Pull request by Juanchimienti**
**What this PR does / why we need it**:

This is a new helm chart for the prometheus mysql exporter 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by unguiculus** [comment]
Please make sure you compare with what you get from `helm create` using Helm 2.8+ and update accordingly.

File: stable/prometheus-mysql-exporter/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## TL;DR;
+
+```console
+$ helm install stable/prometheus-mysql-exporter
+```
+
+## Introduction
+
+This chart bootstraps a prometheus [mysqk exporter](http://github.com/prometheus/mysql_exporter) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. The exporter can connecto mysql directly or using the [Cloud SQL Proxy](https://cloud.google.com/sql/docs/mysql/sql-proxy).
```
Comment (gianrubio) [inline stable/prometheus-mysql-exporter/README.md]: > can connectto... 

you missed a space here

**Comment by Juanchimienti** [comment]
Hey people, any comments?

File: stable/prometheus-mysql-exporter/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "prometheus-mysql-exporter.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Secret name for cloudsql credentials
+*/}}
+{{- define "prometheus-mysql-exporter.cloudsqlsecret" -}}
```
Comment (unguiculus) [inline stable/prometheus-mysql-exporter/templates/_helpers.tpl:37]: Should be based on the fullname partial. I'd suggest this:

`{{ template "prometheus-mysql-exporter.fullname" }}-cloudsqlsecret`

File: stable/prometheus-mysql-exporter/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      app: {{ template "prometheus-mysql-exporter.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "prometheus-mysql-exporter.name" . }}
+        release: {{ .Release.Name }}
+      annotations:
+      {{- if .Values.cloudsqlproxy.enabled }}
+        checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```
Comment (unguiculus) [inline stable/prometheus-mysql-exporter/templates/deployment.yaml]: The `$` symbol is unnecessary here and could be confusing. Please remove it.

**Review by unguiculus — CHANGES_REQUESTED** [review]
I'd like you to indent lists consistently throughout the chart. Either use zero or two spaces.

**Comment by Juanchimienti** [comment]
Changes applied

**Comment by Juanchimienti** [comment]
@gianrubio can you re add the lgtm ?

File: stable/prometheus-mysql-exporter/templates/_helpers.tpl [hunk 4]
```diff
@@ -30,13 +30,3 @@ Create chart name and version as used by the chart label.
 {{- define "prometheus-mysql-exporter.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
-
-{{/*
-Secret name for cloudsql credentials
-*/}}
-{{- define "prometheus-mysql-exporter.cloudsqlsecret" -}}
```
Comment (unguiculus) [inline stable/prometheus-mysql-exporter/templates/_helpers.tpl]: You should have kept it but changed as I suggested. It is still used in the chart.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by Juanchimienti** [comment]
Sorry @unguiculus I didn't get your suggestion at first.

Is this what you mean ?

**Review by unguiculus — APPROVED** [review]

**Pull request merged**